### PR TITLE
Pnp ps publishingimagerenditions

### PR DESCRIPTION
**Pull request by skaggej**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| New sample?      | no
| Related issues?  | none

#### What's in this Pull Request?

This PR contains a new method called CreatePublishingImageRendition. This new method allows for the simple creation of unique Image Renditions in SharePoint. The new Image Rendition's Name must be unique; if it already exists, it will not be created.

### CODE REVIEW

**Comment by msftgits** [comment]
Hi, I am closing and re-opening this PR to bump the CLA bot. Sorry for the inconvenience!

File: Core/OfficeDevPnP.Core/packages.config [hunk 1]
```diff
@@ -7,6 +7,7 @@
   <package id="Microsoft.Data.Services.Client" version="5.6.2" targetFramework="net45" />
   <package id="Microsoft.Graph" version="1.1.1" targetFramework="net45" />
   <package id="Microsoft.Graph.Core" version="1.2.1" targetFramework="net45" />
+  <package id="Microsoft.Identity.Model.Extensions" version="2.0.1459.0" targetFramework="net45" />
```
Comment (erwinvanhunen) [inline Core/OfficeDevPnP.Core/packages.config]: Please remove this nuget reference from the project as it is references a repacked Microsoft official assembly. The nuget package has been released by an individual not associated with Microsoft. If you remove the nuget package I'm happy to reevaluate the PR. Thanks :-)

Comment (skaggej) [inline Core/OfficeDevPnP.Core/packages.config]: I've removed that reference. I must be missing something...without that Nuget reference, I can't build the solution. I've got the Windows Identity Foundation installed, so I'm not sure what I'm missing. Ideas?

**Comment by erwinvanhunen** [comment]
Thanks Eric! I manually merged your PR due to some conflicts with the dev branch. I also refactored it a bit (moved the method to it's own PublishingExtensions class)

**Pull request merged**